annotation-processing/src/integrationTest/java/com/exactpro/epfast/CreatorTests.java

```diff
@@ -1,27 +1,53 @@
 package com.exactpro.epfast;
```

Comment:
add @FastType annotated class outside of  @FastPackage packages. We need to check that proper fast package name is selected in this case